<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_GL">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,0,4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_matrix">
            <property name="styleSheet">
             <string notr="true">QLabel{  
  font-size:18px;  
  color:white;  
}  </string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxMesh">
            <property name="styleSheet">
             <string notr="true">QCheckBox
{
background-color: red;
color:white;
font-family:Microsoft YaHei;
    height:34px;
    font-size:20px;
    border-radius:2px;
}
QCheckBox::indicator
{
    width:20px;
    height:20px;
}</string>
            </property>
            <property name="text">
             <string>Mesh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>582</width>
               <height>51</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="import1_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>58</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>1000</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
background-color: #1E90FF;
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: #8B7355;
font: bold 16px;
min-width:2em;
color:white;
font-family:Microsoft YaHei;
padding: 5px;
}
</string>
                </property>
                <property name="text">
                 <string>Import1</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="import2_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>58</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>10000</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
background-color: #1E90FF;
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: #8B7355;
font: bold 16px;
min-width:2em;
color:white;
font-family:Microsoft YaHei;
padding: 5px;
}
</string>
                </property>
                <property name="text">
                 <string>Import2</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="coarse_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>58</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>1000</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
background-color: #1E90FF;
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: #8B7355;
font: bold 16px;
min-width:2em;
color:white;
font-family:Microsoft YaHei;
padding: 5px;
}
</string>
                </property>
                <property name="text">
                 <string>Coarse</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="fine_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>58</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>1000</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
background-color: #1E90FF;
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: #8B7355;
font: bold 16px;
min-width:2em;
color:white;
font-family:Microsoft YaHei;
padding: 5px;
}
</string>
                </property>
                <property name="text">
                 <string>Fine</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="fusion_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>58</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>1000</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
background-color: #1E90FF;
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: #8B7355;
font: bold 16px;
min-width:2em;
color:white;
font-family:Microsoft YaHei;
padding: 5px;
}
</string>
                </property>
                <property name="text">
                 <string>Final Alignment</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
